README.md

```diff
@@ -59,41 +52,16 @@ pip to install directly from the master branch on github:
 ```
 pip install git+https://github.com/keras-team/keras-nlp.git --upgrade
 ```
-## Using KerasNLP with Keras Core
-
-As of version `0.6.0`, KerasNLP supports multiple backends with Keras Core out 
-of the box. There are two ways to configure KerasNLP to run with multi-backend 
-support:
-
-1. Via the `KERAS_BACKEND` environment variable. If set, then KerasNLP will be 
-using Keras Core with the backend specified (e.g., `KERAS_BACKEND=jax`).
-2. Via the `.keras/keras.json` and `.keras/keras_nlp.json` config files (which 
-are automatically created the first time you import KerasNLP):
-   - Set your backend of choice in `.keras/keras.json`; e.g., `"backend": "jax"`. 
-   - Set `"multi_backend": True` in `.keras/keras_nlp.json`.
-
-Once that configuration step is done, you can just import KerasNLP and start 
-using it on top of your backend of choice:
-
-```python
-import keras_nlp
-
-gpt2_lm = keras_nlp.models.GPT2CausalLM.from_preset("gpt2_base_en")
-gpt2_lm.generate("My trip to Yosemite was", max_length=200)
-```
-
-Until Keras Core is officially released as Keras 3.0, KerasNLP will use 
-`tf.keras` as the default backend. To restore this default behavior, simply 
-`unset KERAS_BACKEND` and ensure that  `"multi_backend": False` or is unset in 
-`.keras/keras_nlp.json`. You will need to restart the Python runtime for changes 
-to take effect.
 
 ## Quickstart
 
-Fine-tune BERT on a small sentiment analysis task using the 
+Fine-tune BERT on a small sentiment analysis task using the
 [`keras_nlp.models`](https://keras.io/api/keras_nlp/models/) API:
 
 ```python
+import os
+os.environ["KERAS_BACKEND"] = "jax"  # Or "tensorflow", or "torch".
```

Comment:
All our guides and examples show setting `KERAS_BACKEND`, it seems consistent to do so here as well.